### ISSUE TITLE
Fix the testGivenPasswords for encryption method tests

### ISSUE THREAD

**Issue by ljacqu** [Xephi/AuthMeReloaded#375]
`testGivenPasswords` in `AbstractEncryptionMethodTest` are temporarily disabled as they don't always run locally.

I'm not very happy with ignoring a whole load of tests because they don't work locally when they do remotely.

**Comment by sgdc3** [comment]
What's the issue when they are executed locally? Errors, missings libs...?

**Comment by ljacqu** [comment]
I believe when running `mvn install` on Windows console the test fails because of the password with special characters. But it works fine if run with IntelliJ or here on Xephi's Jenkins. Probably we just need to add some command line argument when executing Maven command on console. I think there's a -Dencoding=utf-8 thing or something like that; I haven't checked.

**Comment by sgdc3** [comment]
:/ it should work anyway

**Comment by ljacqu** [comment]
I have the same issue. If I run `mvn test` on Windows console the password with the non-ASCII characters fails, but if I run it inside of IntelliJ the tests are fine. I searched around and tried some command line parameters but none can fix the issue. I've found similar accounts online with no solution, sadly.
I'll revert the `@Ignore` since the tests work fine in the "important environments." If it proves to be too much of a problem we'll have to see if we can set up a Maven profile or something? But ultimately we should recommend users to forbid non-ASCII characters in the password anyway, exactly because of such issues. (Not only on our side but their database/file system might mess up.)
